src/Symfony/Component/PropertyAccess/PropertyAccessor.php

```diff
@@ -414,12 +424,21 @@ private function readProperty(array $zval, string $property, bool $ignoreInvalid
                         throw $e;
                     }
                 } elseif (PropertyReadInfo::TYPE_PROPERTY === $type) {
-                    if (!isset($object->$name) && !\array_key_exists($name, (array) $object)) {
+                    $valueSet = false;
+                    $useBypass = $this->byPassPropertyHooks & self::BYPASS_HOOKS_ON_PROPERTY_READ && $access->hasHook() && !$access->isVirtual();
```

Comment:
You must update the `symfony/property-info` dependency to `^7.3` to be sure to have access to the new methods.

Reply:
2025 goal: more contributions to symfony, so I get used to things like that ! :D